.devcontainer/devcontainer.json

```diff
@@ -59,8 +63,22 @@
 					"clangd.restart",
 				],
 			}
-		]
-	},
+		],
+		"python.analysis.extraPaths": [
+			"/workspaces/magma/orc8r/gateway/python/",
+			"/workspaces/magma/lte/gateway/python/",
+		],
+		"python.defaultInterpreterPath": "/usr/bin/python",
+        "python.formatting.provider": "autopep8",
+		"python.formatting.autopep8Args": [
+			// This should be the same set of flags as ones specified in `lte/gateway/precommit.py`
+			"--select=W191,W291,W292,W293,W391,E131,E2,E3",
+		],
+        "[python]": {
+          "editor.formatOnSave": true,
```

Comment:
I guess we do that in this section so that non-python files are not autoformatted. Do you know why we don't do the same in the vscode config for the VM?

Reply:
Yes, exactly. Not sure why we don't have it in the VM. I did this because a language agnostic formatOnSave had effects that didn't want (for example when working on the Dockerfile of the devcontainer it introduces unnecessary changes).